app/templates/search/_pagination.html

```diff
@@ -1,30 +1,24 @@
-<nav role="navigation">
-    <ul class="previous-next-navigation">
+{% if not dmPreviousNextPagination %}
+  {% from "digitalmarketplace/components/previous-next-pagination/macro.njk" import dmPreviousNextPagination %}
+{% endif %}
```

Comment:
I'm not sure the import guard is necessary

Reply:
I thought so, but it ends up failing this test with a `'?live-results=true'` arg:

https://github.com/alphagov/digitalmarketplace-buyer-frontend/blob/37f0d7da87b41843f809c107b7cd376d6487308f/tests/main/views/test_marketplace.py#L1794-L1800

I think it's related to this bit here which seems to only be rendering the parts it needs, so it doesn't catch the base page?
https://github.com/alphagov/digitalmarketplace-buyer-frontend/blob/37f0d7da87b41843f809c107b7cd376d6487308f/app/main/views/marketplace.py#L349-L373

Reply:
Ah, alright then. Thanks for explaining that.